### ISSUE TITLE
no support for win64

### ISSUE THREAD

**Issue by abdulwajid725** [uiureo/node-screencapture#14]
script takes screenshot of whole screen in win64..NOT interactive as in linux..please have a look at it.

**Comment by uiur** [comment]
ref: https://github.com/uiureo/node-screencapture/pull/4#issuecomment-139939773

**Comment by fritx** [comment]
yeah it was because the fallback solution for windows system: `nircmd.exe` doesn't support interactive selection. :/
